learning.py

```diff
@@ -298,6 +298,18 @@ def sample(self):
             self.sampler = weighted_sampler(list(self.dictionary.keys()),
                                             list(self.dictionary.values()))
         return self.sampler()
+        
+    def remove_sparse(self,sparse_value):
+    	""" keep items with probablity sparse value 
```

Comment:
Capitalize comment start. Also, it should be 'Keep items with probability greater than sparse value' or something like that.